armi/settings/fwSettings/globalSettings.py

```diff
@@ -529,7 +530,7 @@ def defineSettings() -> List[setting.Setting]:
             CONF_FRESH_FEED_TYPE,
             default="feed fuel",
             label="Fresh Feed Type",
-            description="None",
+            description="The type of fresh fuel added to the core.",
```

Comment:
Can we add some more context to this description? For instance, when and how it is used? I am afraid that people will see this setting and think that they need to use it, when in reality hardly anybody ever needs to or will use this setting.

Reply:
See what you think.

Reply:
I made a recommendation, see https://github.com/terrapower/armi/pull/1761#discussion_r1663363137

Reply:
done